pkgs/by-name/cl/cloak-pt/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule {
+  pname = "Cloak";
+  version = "2.8.0";
+
+  src = fetchFromGitHub {
+    owner = "BANanaD3V"; # Actual upstream (cbeuw) got a go.mod issue so we use this
```

Comment:
Please use upstream repo with a proper patch if required.

Reply:
Sure. We got some discussion going in the PR i sent to the upstream already.